workers/data_refinery_workers/downloaders/sra.py

```diff
@@ -306,7 +305,7 @@ def download_sra(job_id: int) -> None:
     try:
         if _has_unmated_reads(sample.accession_code, job):
             original_files = _replace_dotsra_with_fastq_files(sample, job, original_file)
-    except:
+    except ftplib.all_errors:
```

Comment:
You also need to import `ftplib`.

Reply:
I think I imported it on the first line but the file view hides it

Reply:
I guess it was already imported? I checked and it is in fact on line 1.

Reply:
Oh weird